### ISSUE TITLE
VS2015 Support and MSBuild 14

### ISSUE THREAD

**Issue by bchavez** [bau-build/bau-msbuild#21]
Hi,

In order to get MSBuild 14 (shipped with VS2015) the `MSBuildVersion` is semantically incorrect ie:(VS14 which doesn't exist).

```
.Do(msb =>
    {
        msb.ToolsVersion = "14.0";
        msb.MSBuildVersion = "VS14"; //Hack for MSBuild VS2015

        msb.Solution = Projects.SolutionFile.ToString();
        msb.Properties = new
            {
                Configuration = "Release",
                //OutDir = Folders.CompileOutput
            };
        msb.Targets = new[] {"Rebuild"};
    })
```

**Comment by adamralph** [comment]
@bchavez thanks for raising this.

**Comment by bchavez** [comment]
I can submit a pull request if you prefer. It should be an easy fix.

**Comment by adamralph** [comment]
Sure, please go ahead!

For reference - https://github.com/bau-build/bau-msbuild/blob/master/CONTRIBUTING.md

**Comment by adamralph** [comment]
@bchavez I've released 0.1.0-beta02 which contains this change.

**Comment by bchavez** [comment]
awesome. thank you! :+1:
